### ISSUE TITLE
`route_after_login` is not documented

### ISSUE THREAD

**Issue by jqnatividad** [ckan/ckan#7084]
**CKAN version**
master

**Describe the bug**
The `ckan.route_after_login` configuration option is not documented.

**Expected behavior**
The option is documented with default and valid values.

**Additional details**
see https://github.com/ckan/ckan/pull/4770

**Comment by amercader** [comment]
Yes, and it should follow the same naming convention as other similar config options like [`ckan.auth.login_view`](https://docs.ckan.org/en/latest/maintaining/configuration.html#ckan-auth-login-view), so it should be named `ckan.auth.route_after_login` (but support the old version of course)

**Comment by jqnatividad** [comment]
Hi @amercader , since it was an undocumented option, should we even support the old configuration option name?

Perhaps, when documenting it, instead mention the old name?

**Comment by amercader** [comment]
I always lean towards trying to not break stuff, specially if the config option has been around for a while. We can show a deprecation message if the old variant is used and drop it after a ckan version
